packages/devtools_app/lib/src/memory/memory_allocation_table_data.dart

```diff
@@ -0,0 +1,159 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../table_data.dart';
+import 'memory_protocol.dart';
+
+const fieldWidthNumbers = 100.0;
+
+class FieldClassName extends ColumnData<ClassHeapDetailStats> {
+  FieldClassName() : super('Class');
+
+  @override
+  String getValue(ClassHeapDetailStats dataObject) => dataObject.classRef.name;
+
+  @override
+  String getDisplayValue(ClassHeapDetailStats dataObject) =>
+      '${getValue(dataObject)}';
+
+  @override
+  bool get supportsSorting => true;
+
+  @override
+  int compare(ClassHeapDetailStats a, ClassHeapDetailStats b) {
+    final Comparable valueA = getValue(a);
+    final Comparable valueB = getValue(b);
+    return valueA.compareTo(valueB);
+  }
+
+  @override
+  double get fixedWidthPx => 200.0;
+}
+
+class FieldInstanceCountColumn extends ColumnData<ClassHeapDetailStats> {
+  FieldInstanceCountColumn()
+      : super(
+          'Instances',
+          alignment: ColumnAlignment.right,
+        );
+
+  @override
+  dynamic getValue(ClassHeapDetailStats dataObject) =>
+      dataObject.instancesCurrent;
+
+  @override
+  String getDisplayValue(ClassHeapDetailStats dataObject) =>
+      '${getValue(dataObject)}';
+
+  @override
+  bool get numeric => true;
+
+  @override
+  bool get supportsSorting => true;
+
+  @override
+  int compare(ClassHeapDetailStats a, ClassHeapDetailStats b) {
+    final Comparable valueA = getValue(a);
+    final Comparable valueB = getValue(b);
+    return valueA.compareTo(valueB);
+  }
+
+  @override
+  double get fixedWidthPx => fieldWidthNumbers;
+}
+
+class FieldInstanceAccumulatorColumn extends ColumnData<ClassHeapDetailStats> {
+  FieldInstanceAccumulatorColumn()
+      : super(
+          'Accum',
```

Comment:
Is there another name we can show that is clearer than 'Accum'? Maybe we can use the same terminology as when diffing two snapshots so
Delta or Difference?

Reply:
I like Delta (it's short) I'll use that.  I will only show positive I think that more important, its showing leaks.  Maybe we could have a setting default that only shows growth unless turned off then displays both positive and negative growth?